pyfr/solvers/navstokes/kernels/bcs/sub-out-fp.mako

```diff
@@ -6,7 +6,7 @@
 % for i in range(nvars - 1):
     ur[${i}] = ul[${i}];
 % endfor
-    ur[${nvars - 1}] = ${c['p']/(c['gamma'] - 1)}
+    ur[${nvars - 1}] = (${c['p']})/${(c['gamma'] - 1)}
```

Comment:
Yes, agreed.

On Wed, Jul 15, 2015 at 3:33 PM, Freddie Witherden <notifications@github.com
> wrote:

> In pyfr/solvers/navstokes/kernels/bcs/sub-out-fp.mako
> <https://github.com/vincentlab/PyFR/pull/64#discussion_r34684101>:
>
> > @@ -6,7 +6,7 @@
> >  % for i in range(nvars - 1):
> >      ur[${i}] = ul[${i}];
> >  % endfor
> > -    ur[${nvars - 1}] = ${c['p']/(c['gamma'] - 1)}
> > +    ur[${nvars - 1}] = (${c['p']})/${(c['gamma'] - 1)}
>
> I think after we merge the new space time Characteristic BC that we will
> not need the parenthesis.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vincentlab/PyFR/pull/64/files#r34684101>.
>



-- 
================================
Dr. Iyer Arvind Sundaram

The reasonable man adapts himself to the world; the unreasonable one
persists in trying to adapt the world to himself.

Therefore all progress depends on the unreasonable man

- George Bernard Shaw.